### ISSUE TITLE
chrome android

### ISSUE THREAD

**Issue by carlituxman** [furf/jquery-ui-touch-punch#119]
Hi, I tested the demo page (http://touchpunch.furf.com/) in android only works with android navigator, with chrome for android not work

**Comment by timelf123** [comment]
:+1: We had 2 jquery UI sliders on a page and they were broken in Android stock browser or Chrome on android, but work fine in iPhone 5. 

Behavior: 
interact with slider 1. 
interact with slider 2. 
slider 1 now slides both sliders.

After doing some debugging, we found that the "stop" event is not registered until a second touch of the page, so both sliders move unless you tap on whitespace in the page

**Comment by drelei** [comment]
Helping confirm that on my Galaxy S4, the sortable demo exhibits the following:

chrome: doesn't work
"internet" app: works
dolphin: works

any insight/fix would be great!

**Comment by iurisilvio** [comment]
Same problem here. It doesn't work on Chrome (Nexus 4).

**Comment by njbmartin** [comment]
Yep, same problem. Doesn't work in the new Chrome for Android, but works in other Android browsers. Silly Chrome...

**Comment by timelf123** [comment]
Should this be logged as a Chrome issue?

**Comment by timaschew** [comment]
Opera 16 also not Working on my nexus 4

**Comment by ian-dowhile** [comment]
None of the demos are working in Chrome for Mobile version 30.0.1599.92, on my Nexus 7 (Android version 4.3).

The demos do work in Firefox version 25 on the same device but not in Opera Mini version 7.5.33361.

I don't know anything about getting this working on a mobile device, so I'd really appreciate it if anyone can help?!

**Comment by Molkobain** [comment]
Hi,

I had the same issue, it's because the js files are link directly from github which give a 'text/plain' header and therefore Chrome doesn't load it as Javascript. It's the same on your computer, open the developer console, go to the Network tab and go to the demo page.You will see that it doesn't load the JQuery Touch Punch file.

To resolve this you juste have to put the jsfile on your own server.

**Comment by Molkobain** [comment]
I didn't know about the issue in Opera, as is happening only in Chrome and Opera we can assume that this is a Webkit issue.

**Comment by sebastianzillessen** [comment]
Hello. For me a list of tries on my Nexus 4 and Nexus 10 with installed Android 4.4 (KitKat):
- Chrome (V 31.0.1650.59) : Fail
- Firefox (V 25.0.1): Ok
- Dolphin (V 10.1.2) : Fail

I'm hoping this issue gets fixed really fast! I want to include this in my application as fast as possible! Thanks!

**Edit**
I could fix the issue by not trying the example on the web page, but creating a new local version without the github raw code worked fine!

**PLEASE DO NOT USE https://raw.github.com/furf/jquery-ui-touch-punch/master/jquery.ui.touch-punch.min.js IN ANY OF YOUR SCRIPTS! This breaks it!***
